microsite/pages/en/index.js

```diff
@@ -31,6 +31,14 @@ class Index extends React.Component {
                 !
               </div>
             </Banner.Dismissable>
+            <Banner.Dismissable storageKey="2022-03-16-newsletter-cta">
```

Comment:
maybe a different storage key from the other one
```suggestion
            <Banner.Dismissable storageKey="2022-09-01-backstagecon-cta">
```

Reply:
Oh right, good call.